src/platform/forms-system/test/js/utilities/findDuplicateIndexes.unit.spec.js

```diff
@@ -0,0 +1,26 @@
+import { expect } from 'chai';
+
+import findDuplicateIndexes from 'platform/forms-system/src/js/utilities/data/findDuplicateIndexes';
+
+describe('duplicateIndexes', () => {
+  const base = [{ x: 'one' }, { x: 'two' }, { x: 'three' }];
+  it('should not find duplicates', () => {
+    expect(findDuplicateIndexes(base, 'x')).to.have.lengthOf(0);
+  });
```

Comment:
I love these simple unit tests. :grin:

Reply:
I'll add one more to demonstrate case-insensitivity matches